src/apiProxy/api-proxy.js

```diff
@@ -304,6 +311,9 @@ export const makeApiRequest$ = (request, API_TIMEOUT, duotoneUrls) => {
 const apiProxy$ = ({ API_TIMEOUT=5000, baseUrl='', duotoneUrls={} }) => {
 
 	return request => {
+		if (!request.state.oauth_token) {
```

Comment:
Actually I think I can remove this block since it's taken care of by the auth plugin in the `verifyAuth` function.

BUT, yes, the user will get the red error screen (in dev) from the `onPreResponse` handler applied to the server [here](https://github.com/meetup/meetup-web-platform/blob/WP-62_cookie-auth/src/server.js#L58).

Essentially this is an operational error so the request needs to crash.